src/containers/adminPage/searchPanels/SelectionPanelSearch.jsx

```diff
@@ -1,115 +1,107 @@
-import React from 'react'
+import React, { useState } from 'react'
 import ReactAutosuggest from 'react-autosuggest'
 import debounce from 'lodash.debounce'
 
 import service from '../../../service'
 import './styles.scss'
 
-class SelectionPanelSearch extends React.Component {
-    constructor() {
-        super()
-        this.state = {
-            value: '',
-            suggestions: [],
-        }
-    }
+const SelectionPanelSearch = ({ handleAddNewStop }) => {
+    const [value, setValue] = useState('')
+    const [suggestions, setSuggestions] = useState([])
 
-    onChange = (event, { newValue, method }) => {
+    const onChange = (_, { newValue, method }) => {
         if (method === 'click') {
-            this.setState({
-                value: '',
-            })
+            setValue('')
         } else {
-            this.setState({
-                value: newValue,
-            })
+            setValue(newValue)
         }
     }
 
-    getSuggestions = (value) => {
+    /* const getSuggestions = value => {
```

Comment:
Why is this method commented out?

Reply:
Yes! It was not in use that's why I commented it instead of deleting. Also, Previously in the class component, there were some state fields that are not initiated but are changed in the methods and there were some that are not in use.

Reply:
Yes, we should go through and clean up a bit. Thanks!